index.html

```diff
@@ -0,0 +1,228 @@
+                        <li id="hamburger"><a><i class="bi bi-list"></i></a></li>
+                        <li class="nav"><a href="#"><i class="bi bi-twitter"></i></a></li>
+                        <li class="nav"><a href="#"><i class="bi bi-facebook"></i></a></li>
+                        <li class="nav"><a class="upper-nav-anchor" href="#"><span>My
+                                    Page</span></a></li>
+                        <li class="nav"><a href="#"><span>English</span></a></li>
+                        <li class="nav"><a href="#"><span>Login</span></a></li>
+                    </ul>
+                </nav>
+                <nav>
+                    <ul id="lower-nav" class="nav">
+                        <li><img src="images/logo.png" alt="Dev Courses Logo"
+                                width="70"></li>
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="#">Older</a></li>
+                        <li><a href="#">Trending</a></li>
+                        <li><a href="#">Your Couses</a></li>
+                    </ul>
+                </nav>
+                <nav id="mobile-open-nav" class="display-none">
+                    <ul>
+                        <li id="close-button"><img
+                                src="images/Icon-close.png"
+                                alt="Close button"></li>
+                        <li><a href="#">Login</a></li>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">My Page</a></li>
+                        <li><a href="#">Older</a></li>
+                        <li><a href="#">Trending</a></li>
+                        <li><a href="#">Your Couses</a></li>
+                        <li><a href="#"><i class="bi
+                                    bi-twitter"></i></a></li>
+                        <li><a href="#"><i class="bi bi-facebook"></i></a></li>
+                        <li><a href="/about.html">About</a></li>
+                        <li><a href="#">English</a></li>
+                    </ul>
+                </nav>
+                <section id="header-content">
+                    <h2 id="hello-header" class="orange-color">"Hello! Fellow
+                        Developers"</h2>
+                    <h1 class="orange-color">A LIST OF THE MOST USEFUL COURSES<br>
+                        FOR DEVELOPERS IN ALL
+                        FIELDS
+                        IN<br>
+                        2021</h1>
+                    <p>
+                        Browse through the website for the list of the greatest
+                        courses
+                        created in programming. Join our live webinars and start
+                        learning at
+                        your own pace with some of the highest quality courses
+                        ever
+                        created
+                        in programming.
+                    </p>
```

Comment:
- [ ] Your description text under the main header text has a longer width or is unspecified. Kindly note that we actually want our design to as much as possible follow the provided [design template](![image](https://user-images.githubusercontent.com/12745474/132543727-b3a11c5a-3eeb-4f35-b53c-8fe6867fa07c.png)). I therefore urge you to kindly improve upon this section by defining a width for this text so that it closely matches the length as seen in the provided template design.

**Your top section:**
![image](https://user-images.githubusercontent.com/12745474/132543768-140a00c8-cb39-42bd-9bf4-d99b2ca61715.png)

**The original design top section:**
![image](https://user-images.githubusercontent.com/12745474/132544021-b96c7535-24be-40b0-a1fc-282726586dc2.png)